cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -18,14 +18,20 @@ package app
 
 import (
 	"net/http"
+	"time"
 
+	arktos "k8s.io/arktos-ext/pkg/generated/clientset/versioned"
+	"k8s.io/arktos-ext/pkg/generated/informers/externalversions"
+	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/klog"
 	controllers "k8s.io/kubernetes/pkg/controller/mizar"
 )
 
 const (
-	mizarStarterControllerWorkerCount = 2
-	mizarPodControllerWorkerCount     = 4
+	mizarStarterControllerWorkerCount       = 2
+	mizarPodControllerWorkerCount           = 4
+	mizarServiceControllerWorkerCount       = 4
+	mizarArktosNetworkControllerWorkerCount = 4
 )
 
```

Comment:
As our project is open-sourced, we can put a link to the design doc about mizar-arktos-integration in the beginning of this file, to help those code readers outside Futurewei.